File: Programacion/Tema 10/Ejercicios/Ejercicios 1/Ejercicios 1/src/Ejercicio_1/Empleado.java
package Ejercicio_1;

public class Empleado {
    private String nombre;
    private double salario;

    public Empleado(String nombre, double salario) {
        this.nombre = nombre;
        this.salario = salario;
    }
    public String mostrarinfo(){
        return "nombre "+nombre+" salario "+salario;
    }
}

File: Programacion/Tema 7/Ejercicios/Repaso Examen/Repaso_Examen/src/main/kotlin/Empleado.kt
class Empleado(
    val nombre: String,
    val edad:Int
)
fun calcularjubilacion(empleado: Empleado){
    val calculo=65-empleado.edad
    if (calculo>0){
        println("Te quedan $calculo años para jubilarte")
    }else{
        println("Ya estas jubilado")
    }
}